server/komodo-server/src/main/java/org/komodo/rest/service/KomodoUtilService.java

```diff
@@ -1233,4 +1168,8 @@ public Response removeViewEditorState(final @Context HttpHeaders headers,
             return createErrorResponseWithForbidden(mediaTypes, e, RelationalMessages.Error.PROFILE_EDITOR_STATE_REMOVE_ERROR);
         }
     }
+    
```

Comment:
can remove - dont think this is used anymore

Reply:
done